packages/react-progress-bar/lib/react-progress-bar.js

```diff
@@ -30,13 +30,20 @@ export const ProgressBar = ({
 		</span>
 	);
 
+	const checkValue = (value) => {
+		if('undefined' !== typeof value && null !== value && !isNaN(value)) {
```

Comment:
@creador-dev, on line 17, we already check if the `now` property value is a number or undefined but later, with the `checkValue` function, we check for it again. Let's better not be redundant.